src/makeInfinityAdder.js

```diff
@@ -4,7 +4,23 @@
  * @return {function}
  */
 function makeInfinityAdder() {
-  // write code here
+  let sumOfAllNumbers = 0;
+  let copyOfSum;
+
+  const adder = (number = null) => {
+    sumOfAllNumbers += number;
```

Comment:
There is a bug in this line. You are adding `number` to `sumOfAllNumbers` even if `number` is `null`. This results in `NaN` if `adder` is called without any parameters. You should add `number` to `sumOfAllNumbers` only if `number` is not `null`.